test/runtests.jl

```diff
@@ -143,6 +143,10 @@ const url = "http://127.0.0.1:$port"
                 req = HTTP.post("$url/debug_engine", headers, payload)
                 @test req.status == 200
                 fname = read(IOBuffer(req.body), String)
+                @static if isdefined(Profile, :HeapSnapshot) && isdefined(Profile.HeapSnapshot, :assemble_snapshot) && Sys.isunix()
+                    # test whether the returned file has a tar extension
+                    @test occursin(".tar", fname)
+                end
```

Comment:
@d-netto: This test was added to the wrong test-function. It should have been for "heap snapshot," not "allocation profile."
I'll fix it in https://github.com/JuliaPerf/ProfileEndpoints.jl/pull/50.